content/_data/logo/googly.yml

```
@@ -0,0 +1,7 @@
+---
+name: 'Googly'
+url: '/images/logos/googly/googly.png'
+url_256: '/images/logos/googly/256.png'
+vector: '/images/logos/googly/googly.svg'
```

Don't need the /images/ prefix
```suggestion
url: 'logos/googly/googly.png'
url_256: 'logos/googly/256.png'
vector: 'logos/googly/googly.svg'
```